lib/network/api/node_info_test.go

```diff
@@ -0,0 +1,115 @@
+package api
+
+import (
+	"bufio"
+	"encoding/json"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"boscoin.io/sebak/lib/block"
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/storage"
+	"boscoin.io/sebak/lib/version"
+	"github.com/gorilla/mux"
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPIGetNodeInfoHandler(t *testing.T) {
+	st := storage.NewTestStorage()
```

Comment:
I think you could save yourself some trouble by just using `block.InitTestBlockchain()`